__tests__/api.test.js

```diff
@@ -308,7 +306,91 @@ const pageCount = [
     })
   }),
 
+  describe('POST /api/articles/:article_id/comments', () => {
+    describe('successful connection test(s)', () => {
+      const comment1 = {
+        body: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit',
+        author: 'lurker',
+      }
+      test('201: page returns with an object', () => {
+        return request(app)
+          .post('/api/articles/1/comments')
+          .send(comment1)
+          .expect(201)
+          .then(({
+            body
+          }) => {
+            expect(typeof body).toBe('object')
+          })
+      })
+      test('201: returns posted comment', () => {
 
+        return request(app)
+          .post('/api/articles/1/comments')
+          .send(comment1)
```

Comment:
please avoid reusing pre-created objects, they should really be done in each test